<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
	background-color: rgb(170, 255, 255)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>411</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TODO PRINTER APP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="printTodoBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_2{
	background-color: rgb(170, 255, 127);
	color: indigo;
	border-radius: 20px;
}

#pushButton_2:hover{
	background: rgb(170, 200, 127);
	color: #3ff;
}</string>
    </property>
    <property name="text">
     <string>PRINT AND DO IT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Quicksand Light</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select Printer</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>560</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Quicksand SemiBold</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>411</width>
      <height>411</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="newTodoBtn">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Quicksand</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton{
	border: none;
	border-radius: 10px;
	background-color: pink;
}

#pushButton:hover{
	background-color: white;
	color: purple;
}</string>
     </property>
     <property name="text">
      <string>+ NEW</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="todoTableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>391</width>
       <height>361</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#tableWidget{
	border: none;
	background: red;
	border-radius: 12px;
}</string>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Label</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>LISTING</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>411</width>
      <height>411</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>LONG TASK </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="frame2TaskName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>391</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Quicksand Light</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Task Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Quicksand Light</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="frame2TaskDescription">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>391</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Quicksand Light</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select Task Type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="taskType">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Quicksand SemiBold</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <item>
     <property name="text">
      <string>LIST</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LONGTASK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NOEMSPRINT</string>
     </property>
    </item>
   </widget>
   <widget class="QFrame" name="frame3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>411</width>
      <height>411</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="newItemBtn">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Quicksand</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton{
	border: none;
	border-radius: 10px;
	background-color: pink;
}

#pushButton:hover{
	background-color: white;
	color: purple;
}</string>
     </property>
     <property name="text">
      <string>ADD NEW ITEM</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="frame3ItemTbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>391</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#tableWidget{
	border: none;
	background: red;
	border-radius: 12px;
}</string>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>QTY</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ITEM NAME</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PRICE</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>TOTAL</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>RECIEPT</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="frame3RecieptName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Quicksand Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>RECIPE NAME</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="frame3ItemName">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Quicksand Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ITEM NAME</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Quicksand Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Quantity</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>150</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Quicksand Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="frame3Qty">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="frame3Price">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>150</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>390</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Quicksand Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TOTAL AMOUNT</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="frame3TotalAmount">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>380</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
    <widget class="QPushButton" name="frame3ResetBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Quicksand</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton{
	border: none;
	border-radius: 10px;
	background-color: pink;
}

#pushButton:hover{
	background-color: white;
	color: purple;
}</string>
     </property>
     <property name="text">
      <string>RESET FIELD</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
